src/test/java/com/epam/tc/hw6/AttachmentUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw6;
+
+import io.qameta.allure.Allure;
+import java.io.InputStream;
+//import lombok.experimental.UtilityClass;
+
+//@UtilityClass
```

Comment:
Please remove commented imports and annotations if they are unnecessary

Reply:
ok